console/src/main/java/com/alibaba/nacos/console/security/nacos/users/NacosUserDetailsServiceImpl.java

```diff
@@ -91,7 +91,7 @@ public void updateUserPassword(String username, String password) {
     
     public User getUser(String username) {
         User user = userMap.get(username);
-        if (!authConfigs.isCachingEnabled()) {
+        if (!authConfigs.isCachingEnabled() || user == null) {
```

Comment:
same as NacosRoleServiceImpl.java

Reply:
> same as NacosRoleServiceImpl.java

OK, I will remove it later and open a new issue to fix it